driver/tke_driver.go

```diff
@@ -159,7 +159,6 @@ func NewDriver() types.Driver {
 	driver.driverCapabilities.AddCapability(types.SetVersionCapability)
```

Comment:
SetVersionCapability is available here, but the version field is not addressed in update function, is it updatable in TKE?